server/src/main/proto/spine/server/delivery/delivery.proto

```diff
@@ -107,3 +108,24 @@ message WorkerId {
     // An example of such an identifier could be ID of the thread which processes a shard.
     string value = 2 [(required) = true];
 }
+
+// An outcome of the `pickUp()` method of method of `ShardedWorkRegistry` indicating
```

Comment:
Please describe the message in terms of domain language, not in terms of some Java API.